src/config.ts

```diff
@@ -38,13 +37,22 @@ export default {
     // Push
     PUSH_URL: 'https://push-web.threema.ch/push',
 
-    // Very verbose logging that potentially affects performance and may also
-    // contain sensitive information.
-    VERBOSE_DEBUGGING: false,
-    // Logs all incoming and outgoing protocol messages.
-    MSG_DEBUGGING: false,
-    // Logs URLs to visualise MsgPack messages for all incoming and outgoing
-    // protocol messages.
-    MSGPACK_DEBUGGING: false,
+    // Console log level
+    CONSOLE_LOG_LEVEL: 'info',
+    // Compose area log level
+    COMPOSE_AREA_LOG_LEVEL: 'warn',
+    // SaltyRTC log level
+    SALTYRTC_LOG_LEVEL: 'warn',
+    // Toggles logging verbose timer-related information.
+    DEBUG_TIMER: false,
+    // Toggles logging all chunks and messages exchanged by or associated with
+    // the app remote protocol. May also enable additional debug facilities of
+    // the app remote protocol.
+    // Note: Affects performance and contains sensitive information.
+    DEBUG_ARP: false,
+    // Toggles URL logging to visualise MsgPack messages for all incoming and
+    // outgoing protocol messages.
+    // Note: Affects performance and contains sensitive information.
+    DEBUG_MSGPACK: false,
```

Comment:
Should we split this up into two objects?

```ts
LOG_LEVEL: {
  CONSOLE: 'info',
  COMPOSE_AREA: 'warn',
  SALTYRTC: 'warn',
},
DEBUG: {
  TIMER: false,
  ARP: false,
  MSGPACK: false,
}
```

I think that would make it much easier to read when adding more entries. On the other hand patching the config is much harder because the values are namespaced on another line of code...

Reply:
My idealistic mind agrees but my practical mind thinks that the config is still well readable atm.

Reply:
Let's keep it, because of the config-patching issue :slightly_smiling_face: